cosmos/uptick_117.json

```diff
@@ -28,16 +29,18 @@
 	"currencies": [{
 		"coinDenom": "UPTICK",
 		"coinMinimalDenom": "auptick",
-		"coinDecimals": 18
+		"coinDecimals": 18,
+		"coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/uptick_117/auptick.png"
 	}],
 	"feeCurrencies": [{
 		"coinDenom": "UPTICK",
 		"coinMinimalDenom": "auptick",
-		"coinDecimals": 18,
+		"coinDecimals": 18,
+		"coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/uptick_117/auptick.png",
 		"gasPriceStep": {
-			"low": 10000000000,
-			"average": 20000000000,
-			"high": 40000000000
+			"low": 125000000000,
+			"average": 130000000000,
+			"high": 150000000000
 		}
 	}],
 	"coinType": 60,

```

Comment:
@xianxianzhou Could you remove `coinType`field?

Reply:
I have remove coinType now